pkg/kubectl/cmd/get.go

```diff
@@ -168,6 +168,7 @@ func RunGet(f cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args [
 	selector := cmdutil.GetFlagString(cmd, "selector")
 	allNamespaces := cmdutil.GetFlagBool(cmd, "all-namespaces")
 	showKind := cmdutil.GetFlagBool(cmd, "show-kind")
+	showAll := cmdutil.GetFlagBool(cmd, "show-all")
```

Comment:
Discussed with @pwittrock, let's don't specialize `--show-all` for `kubectl get`. That is, `kubectl get pods --show-all` doesn't include uninitialized objects. Users have to use `kubectl get pods --include-uninitialized=true`.

The root problem is that `kubectl get` doesn't have a `--all` flag, while many other kubectl commands have. We need to unify the flags, but that's another PR.